music21/musicxml/xmlToM21.py

```diff
@@ -1654,7 +1661,8 @@ def separateOutPartStaves(self):
         '''
         Take a `Part` with multiple staves and make them a set of `PartStaff` objects.
         '''
-
+        # Elements in these classes appear only on the staff to which they are assigned.
+        # All other classes appear on every staff.
```

Comment:
This may not be strictly true -- spanners continue to appear only on the first staff, but the references are valid when manipulating the other staves and makeNotation will gather as needed. The reason `Spanner` is in `STAFF_SPECIFIC_CLASSES` is so that we don't copy them into the template.